packages/frontend/src/contexts/Ceremony.js

```diff
@@ -54,7 +54,7 @@ export default class Queue {
     const hashText = Object.entries(this.contributionHashes ?? {})
       .map(([circuitName, hash]) => `${circuitName}: ${hash}`)
       .join('\n\n')
-    return `I just contributed to the unirep dev trusted setup ceremony. You can too [here](https://ceremony.unirep.io).
+    return `I just contributed to the "Unirep" trusted setup ceremony. You can too [here](https://ceremony.unirep.io).
```

Comment:
I just contributed to the "Unirep" trusted setup ceremony. You can too [here](https://ceremony.unirep.io).

Reply:
oh you mean to replace the `here` to a hyperlink? let me see how to do it, since it looks like gist post is not accepting markdown format.

Reply:
no....
you had "" in the writing. and the case is not matching the brand.